about.html

```diff
@@ -47,22 +47,33 @@ <h2>Technology</h2>
         <!-- begin Funding section -->
         <hr class="whitespace">
         <div class="row column text-center">
-            <h2>Funding and Project Participants</h2>
+            <h2><a class="anchor" id="funding"></a>Funding and Project Participants</h2>
         </div>
         <hr>
         <div class="row">
             <div class="small-10 small-centered medium-10 medium-centered columns">
                 <div class="row horizontal">
                     <div>
-                        <p>This project is funded by the BBSRC
-                        (<a href="https://bbsrc.ukri.org/research/grants/grants/AwardDetails.aspx?FundingReference=BB/R015384/1">BBR Award BB/R015384/1</a>)
-                        and is a collaboration between the
-                        <a href="https://www.openmicroscopy.org">OME Consortium</a> at
-                        Dundee (www.openmicroscopy.org) and EMBL-EBI.
-                        The IDR Project PIs are
+                        <p>The IDR Project PIs are
```

Comment:
This `<p>` doesn't have a corresponding `</p>`

Reply:
Resolved in 3c41ff3